### PR TITLE
WIP:EOS-13829:Logrotate exiting abnormally due to directory permissions issue

### PR DESCRIPTION
**Pull request by sradhanand-pati**
Signed-off-by: sradhanand.pati <sradhanand.pati@seagate.com>

### CODE REVIEW

File: srv/components/system/logrotate/files/etc/logrotate_hw.d/provisioner.conf [hunk 1]
```diff
@@ -9,6 +9,7 @@
 /var/log/seagate/provisioner/storage_metadata_reset.log
 /var/log/seagate/provisioner/salt_master.log
 /var/log/seagate/provisioner/salt_minion.log {
+    su root prvsnrusers
```
Comment (andkononykhin2) [inline srv/components/system/logrotate/files/etc/logrotate_hw.d/provisioner.conf:12]: why we need that here?

File: srv/components/system/logrotate/files/etc/logrotate_hw.d/provisioner.conf [hunk 2]
```diff
@@ -18,6 +19,7 @@
 }
 
 /var/log/salt/key {
+    su root prvsnrusers
```
Comment (andkononykhin2) [inline srv/components/system/logrotate/files/etc/logrotate_hw.d/provisioner.conf:22]: why we need that here?

**Review by andkononykhin2 — COMMENTED** [review]
@sradhanand-pati I believe you find a good solution. But some changes might be not required actually. Please check comments.

One more question: why you renamed these files?

**Comment by ypise** [comment]
> @sradhanand-pati I believe you find a good solution. But some changes might be not required actually. Please check comments.
> 
> One more question: why you renamed these files?

@andkononykhin2 
The files were renamed  as the reporter has mentioned in JIRA that the files are getting duplicated.

@sradhanand-pati 
Rename `srv/components/system/logrotate/files/etc/logrotate_hw.d/provisioner.conf`
```
git mv srv/components/system/logrotate/files/etc/logrotate_hw.d/provisioner.conf srv/components/system/logrotate/files/etc/logrotate_hw.d/provisioner  
```

File: srv/components/system/logrotate/files/etc/logrotate_hw.d/provisioner.conf (hunk 1 above)
Comment (sradhanand-pati) [reply srv/components/system/logrotate/files/etc/logrotate_hw.d/provisioner.conf:12]: i have answered over the group chat

File: srv/components/system/logrotate/files/etc/logrotate_hw.d/provisioner.conf (hunk 2 above)
Comment (sradhanand-pati) [reply srv/components/system/logrotate/files/etc/logrotate_hw.d/provisioner.conf:22]: i have answered over the group chat.
please discuss here if you have ore queries

**Review by ypise — APPROVED** [review]

**Pull request merged**